Only the skins2 bit is necessary, all other changes are already present in VLC
2.2.4.

From a1e5e6422d04cd3380bc5b10c86713fef0832aa3 Mon Sep 17 00:00:00 2001
From: Tristan Matthews <le.businessman@gmail.com>
Date: Wed, 2 Jul 2014 17:28:02 -0400
Subject: [PATCH] Fix build for C++11

C++11 requires a space between literal and identifier
---
 modules/gui/qt4/components/extended_panels.cpp | 4 ++--
 modules/gui/skins2/src/dialogs.cpp             | 2 +-
 modules/video_filter/atmo/AtmoLiveView.cpp     | 4 ++--
 modules/video_filter/atmo/atmo.cpp             | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

--- modules/gui/skins2/src/dialogs.cpp
+++ modules/gui/skins2/src/dialogs.cpp
@@ -212,7 +212,7 @@ void Dialogs::showChangeSkin()
 void Dialogs::showPlaylistLoad()
 {
     showFileGeneric( _("Open playlist"),
-                     _("Playlist Files|"EXTENSIONS_PLAYLIST"|"
+                     _("Playlist Files|" EXTENSIONS_PLAYLIST "|"
                        "All Files|*"),
                      showPlaylistLoadCB, kOPEN );
 }
